### PR TITLE
Fix licenses

### LINKED ISSUE #56: Add license checking github action
https://github.com/SymbiFlow/actions/tree/main/checks

### PR DESCRIPTION
**Pull request by acomodi**
fixes #56

### CODE REVIEW

File: fasm2bels/database/connection_database_cache.py [hunk 1]
```diff
@@ -1,3 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
```
Comment (mithro) [inline fasm2bels/database/connection_database_cache.py:10]: Missing newline here.

File: fasm2bels/database/create_channels.py [hunk 2]
```diff
@@ -1,3 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
```
Comment (mithro) [inline fasm2bels/database/create_channels.py:10]: Missing newline here.

File: fasm2bels/fasm2bels.py [hunk 3]
```diff
@@ -1,3 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
```
Comment (mithro) [inline fasm2bels/fasm2bels.py:10]: Missing newline here.

File: fasm2bels/lib/parse_route.py [hunk 4]
```diff
@@ -1,3 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
```
Comment (mithro) [inline fasm2bels/lib/parse_route.py:10]: Missing newline here.

File: fasm2bels/lib/parse_xdc.py [hunk 5]
```diff
@@ -1,3 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
```
Comment (mithro) [inline fasm2bels/lib/parse_xdc.py:10]: Missing newline here.

File: fasm2bels/models/verilog_modeling.py [hunk 6]
```diff
@@ -1,3 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
```
Comment (mithro) [inline fasm2bels/models/verilog_modeling.py:10]: Missing newline here.

File: fasm2bels/net_map.py [hunk 7]
```diff
@@ -1,3 +1,13 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
```
Comment (mithro) [inline fasm2bels/net_map.py:10]: Missing newline here.

**Review by mithro — COMMENTED** [review]
Just some new lines needed.

Once done, do **not** merge -- Once fixed poke me again so I can add my signoff line to the commits and merge it.

File: fasm2bels/database/connection_database_cache.py (hunk 1 above)
Comment (acomodi) [reply fasm2bels/database/connection_database_cache.py:10]: Actually, yapf complains if a new line is added there. Not sure how to fix it.

I think a possible way would be to move the comment below the imports, or change some yapf configs?

Comment (mithro) [reply fasm2bels/database/connection_database_cache.py:10]: Move the comment below the imports I think?

**Comment by mithro** [comment]
You need to give "maintainers can change commits" access or give me push access to `antmicro/symbiflow-xc-fasm2bels`.

**Review by mithro — APPROVED** [review]
LGTM! Merging now.

**Pull request merged**